index.js

```diff
@@ -36,6 +46,15 @@ Babel.prototype.constructor = Babel;
 Babel.prototype.extensions = ['js'];
 Babel.prototype.targetExtension = 'js';
 
+Babel.prototype.write = function(readTree, destDir) {
+  var self = this;
+  return Filter.prototype.write.call(this, readTree, destDir).then(function() {
+    if (self.exportModuleMetadata) {
+      fs.writeFileSync(destDir + path.sep + 'dep-graph.json', JSON.stringify(self.moduleMetadata, undefined, 2));
```

Comment:
We could technically do a string comparison of each dep-graph if we want to eak out some more perf.